ci/build.yml

```diff
@@ -230,7 +230,7 @@ jobs:
         }
         trap cleanup EXIT
         testConnection() {
-            docker exec oracle bash -c 'sqlplus -L system/hunter2@//localhost:1521/ORCLPDB1 <<< "select * from dba_users;"; exit $?' >/dev/null
+            docker exec oracle bash -c 'sqlplus -L system/'"$ORACLE_PWD"'@//localhost:1521/ORCLPDB1 <<< "select * from dba_users;"; exit $?' >/dev/null
```

Comment:
We could also use double quotes for the outer one and then single quotes for the `select` and escape the `$?`. Not sure that’s clearer, wdyt?

Reply:
```suggestion
            docker exec oracle bash -c <<BASH
                sqlplus -L system/$ORACLE_PWD@//localhost:1521/ORCLPDB1 <<< "select * from dba_users;" >/dev/null
            BASH
```

I think this should also work, though it's arguably a slightly different semantics:

```
docker exec oracle bash -c "sqlplus -L system/$ORACLE_PWD@//localhost:1521/ORCLPDB1" <<< "select * from dba_users;" >/dev/null
```

Reply:
I think the `exit` stuff was required but I forgot why :thinking:

Reply:
Bash returns the exit code of the last command by default AFAIK so I don't think it does anything here. Maybe the command used to be more complex?

Reply:
I thought it was some mix of `docker exec` and `bash -c` that required it but I’ve probably just messed it up at the time.

Reply:
I don't have any specific opinion, but this works and seems fairly readable.